docs/functions/Bundles/Start-UPloadVCFBundle.md

```diff
@@ -1,38 +0,0 @@
-# Start-UploadVCFBundle
-
-### Synopsis
-The Start-UploadVCFBundle starts upload of bundle to SDDC Manager
-
-### Syntax
-```
-Start-UploadVCFBundle -json <path to json file>
-```
-
-### Description
-The Start-UploadVCFBundle cmdlet starts upload of bundle(s) to SDDC Manager
-
-### Examples
-#### Example 1
-```
-Start-UploadVCFBundle -json .\Bundle\bundlespec.json
```

Comment:
Module Name should be 'Start-VCFUploadBundle'

Reply:
It's a deleted file